python/src/filesystem.rs

```diff
@@ -25,7 +24,7 @@ pub(crate) struct FsConfig {
 #[derive(Debug, Clone)]
 pub struct DeltaFileSystemHandler {
     pub(crate) inner: Arc<DynObjectStore>,
-    pub(crate) rt: Arc<Runtime>,
+    pub(crate) rt: Arc<&'static Runtime>,
```

Comment:
Since it's a static, I don't think it even needs to hold this as part of the struct. Can just reference the global static in the methods.

Reply:
@wjones127 yeah this is already changed :P I was committing a bit to fast here

Reply:
Or you mean to not pass it through the struct?

Reply:
@wjones127 I removed it from the structs